### PR TITLE
Update canceling-a-workflow.md

### PR DESCRIPTION
**Pull request by kuboon**
link to `cancelled()` step function

### Why:

They might want to know `cancelled()` function.
I'm not native English speaker. Please brush up my sentences.

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by nguyenalex836** [comment]
@kuboon Thanks so much for opening a PR! I'll get this triaged for review ✨

**Comment by kuboon** [comment]
```
    Rule        search-replace Custom rule
    Detail      docs-domain: Catch occurrences of docs.gitub.com domain.
    Line        31 (col 200) column: 200 text:'docs.github.com'
```
should I change to relative path?

File: content/actions/managing-workflow-runs/canceling-a-workflow.md [hunk 1]
```diff
@@ -28,6 +28,6 @@ When canceling workflow run, you may be running other software that uses resourc
 
 1. To cancel the workflow run, the server re-evaluates `if` conditions for all currently running jobs. If the condition evaluates to `true`, the job will not get canceled. For example, the condition `if: always()` would evaluate to true and the job continues to run. When there is no condition, that is the equivalent of the condition `if: success()`, which only runs if the previous step finished successfully.
 1. For jobs that need to be canceled, the server sends a cancellation message to all the runner machines with jobs that need to be canceled.
-1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run.
+1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run. You can use [cancelled()](https://docs.github.com/en/actions/learn-github-actions/expressions#cancelled) especially for the cancel condition.
```
Comment (kuboon) [inline content/actions/managing-workflow-runs/canceling-a-workflow.md:31]:
```suggestion
1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run. You can use [cancelled()](../learn-github-actions/expressions#cancelled) especially for the cancel condition.
```

File: content/actions/managing-workflow-runs/canceling-a-workflow.md [hunk 2]
```diff
@@ -28,6 +28,6 @@ When canceling workflow run, you may be running other software that uses resourc
 
 1. To cancel the workflow run, the server re-evaluates `if` conditions for all currently running jobs. If the condition evaluates to `true`, the job will not get canceled. For example, the condition `if: always()` would evaluate to true and the job continues to run. When there is no condition, that is the equivalent of the condition `if: success()`, which only runs if the previous step finished successfully.
 1. For jobs that need to be canceled, the server sends a cancellation message to all the runner machines with jobs that need to be canceled.
-1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run.
+1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run. You can use [cancelled()](../learn-github-actions/expressions#cancelled) especially for the cancel condition.
```
Comment (kuboon) [inline content/actions/managing-workflow-runs/canceling-a-workflow.md:31]:
```suggestion
1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run. You can use [cancelled()](/en/learn-github-actions/expressions#cancelled) especially for the cancel condition.
```

File: content/actions/managing-workflow-runs/canceling-a-workflow.md [hunk 3]
```diff
@@ -28,6 +28,6 @@ When canceling workflow run, you may be running other software that uses resourc
 
 1. To cancel the workflow run, the server re-evaluates `if` conditions for all currently running jobs. If the condition evaluates to `true`, the job will not get canceled. For example, the condition `if: always()` would evaluate to true and the job continues to run. When there is no condition, that is the equivalent of the condition `if: success()`, which only runs if the previous step finished successfully.
 1. For jobs that need to be canceled, the server sends a cancellation message to all the runner machines with jobs that need to be canceled.
-1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run.
+1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run. You can use [cancelled()](/en/learn-github-actions/expressions#cancelled) especially for the cancel condition.
```
Comment (kuboon) [inline content/actions/managing-workflow-runs/canceling-a-workflow.md:31]:
```suggestion
1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run. You can use [cancelled()](/en/actions/learn-github-actions/expressions#cancelled) especially for the cancel condition.
```

File: content/actions/managing-workflow-runs/canceling-a-workflow.md [hunk 4]
```diff
@@ -28,6 +28,6 @@ When canceling workflow run, you may be running other software that uses resourc
 
 1. To cancel the workflow run, the server re-evaluates `if` conditions for all currently running jobs. If the condition evaluates to `true`, the job will not get canceled. For example, the condition `if: always()` would evaluate to true and the job continues to run. When there is no condition, that is the equivalent of the condition `if: success()`, which only runs if the previous step finished successfully.
 1. For jobs that need to be canceled, the server sends a cancellation message to all the runner machines with jobs that need to be canceled.
-1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run.
+1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run. You can use [cancelled()](/en/actions/learn-github-actions/expressions#cancelled) especially for the cancel condition.
```
Comment (kuboon) [inline content/actions/managing-workflow-runs/canceling-a-workflow.md:31]:
```suggestion
1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run. You can use [cancelled()](/actions/learn-github-actions/expressions#cancelled) especially for the cancel condition.
```

**Comment by nguyenalex836** [comment]
@kuboon One of our team members can speak to your last question once they review your PR 💛

**Comment by mchammer01** [comment]
I'll review this some time today!

File: content/actions/managing-workflow-runs/canceling-a-workflow.md [hunk 5]
```diff
@@ -28,6 +28,6 @@ When canceling workflow run, you may be running other software that uses resourc
 
 1. To cancel the workflow run, the server re-evaluates `if` conditions for all currently running jobs. If the condition evaluates to `true`, the job will not get canceled. For example, the condition `if: always()` would evaluate to true and the job continues to run. When there is no condition, that is the equivalent of the condition `if: success()`, which only runs if the previous step finished successfully.
 1. For jobs that need to be canceled, the server sends a cancellation message to all the runner machines with jobs that need to be canceled.
-1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run.
+1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run. You can use [cancelled()](/actions/learn-github-actions/expressions#cancelled) especially for the cancel condition.
```
Comment (mchammer01) [inline content/actions/managing-workflow-runs/canceling-a-workflow.md:31]: Suggesting the following, which is in line with our content model.
As for your question on the PR about links, you are correct, we need to use relative links 👍🏻 

```suggestion
1. For jobs that continue to run, the server re-evaluates `if` conditions for the unfinished steps. If the condition evaluates to `true`, the step continues to run. You can use the `cancelled` expression to apply a  status check of `cancelled( )`. For more information see "[AUTOTITLE](/actions/learn-github-actions/expressions#cancelled)."
```

**Review by mchammer01 — COMMENTED** [review]
@kuboon 👋🏻 - thanks for your contribution ✨ 
I've made a suggestion. I will also ask a writer familiar with GitHub Actions for their opinion on this work, and my suggestion. Thanks for your patience with us ❤️

**Review by mchammer01 — APPROVED** [review]
LGTM ✨ 
Got an approval from a team member versed in Actions ⚡

**Pull request merged**